### DISCUSSION TITLE
Upcoming OpenDDS Version 3.29.0

### DISCUSSION THREAD

**Discussion by jrw972** [OpenDDS/OpenDDS#4739] [Announcements]
The OpenDDS Foundation plans to release the master branch as OpenDDS 3.29.0 in the near future. The current release notes can be found [here](https://opendds.readthedocs.io/en/master/news.html#v3-29-0). Any feedback would be greatly appreciated.

**Comment by jwillemsen** [comment]
AXCIOMA builds and runs with current master of OpenDDS and ACE 8.0.0/TAO 4.0.0

**Comment by jwillemsen** [comment]
Support for building with ACE8.0.0/TAO4.0.0 is lacking in the release notes
